### PR TITLE
TF-656 Add request read receipt header

### PR DESCRIPTION
**Pull request by sherlockvn**

### CODE REVIEW

**Review by ManhNTX — COMMENTED** [review]
Add evidence video.

**Review by hoangdat — APPROVED** [review]
LGTM. How about the email in the receiver? Please make the demo with openpaas to make sure it works well. Thanks

File: core/lib/presentation/views/button/icon_button_web.dart [hunk 1]
```diff
@@ -173,4 +173,76 @@ Widget buildButtonWrapText(String name, {
...
+Widget buildIconWithLowerMenu(
+  Icon icon, 
+  BuildContext context, 
+  List<PopupMenuEntry> popupMenuItems, 
+  Function(
+    BuildContext context, 
+    RelativeRect? position, 
+    List<PopupMenuEntry> popupMenuItems
+    ) openPopUpMenuAction, 
+  ) {
```
Comment (dab246) [inline core/lib/presentation/views/button/icon_button_web.dart:187]: Remove 1 indent

File: core/lib/presentation/views/button/icon_button_web.dart [hunk 2]
```diff
@@ -173,4 +173,76 @@ Widget buildButtonWrapText(String name, {
...
+Widget buildIconWithUpperMenu(
+  Icon icon, 
+  BuildContext context, 
+  List<PopupMenuEntry> popupMenuItems, 
+  Function(
+    BuildContext context, 
+    RelativeRect? position, 
+    List<PopupMenuEntry> popupMenuItems
+    ) openPopUpMenuAction, 
+  ) {
```
Comment (dab246) [inline core/lib/presentation/views/button/icon_button_web.dart:223]: same

File: lib/features/composer/presentation/composer_view.dart [hunk 3]
```diff
@@ -141,42 +141,81 @@ class ComposerView extends GetWidget<ComposerController>
...
+                  height: 44,
+                  radius: 10,
+                  onTap: () => controller.sendEmailAction(context)),
+            ]
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              buildIconWithUpperMenu(
+                const Icon(Icons.more_vert),
```
Comment (dab246) [inline lib/features/composer/presentation/composer_view.dart:213]: Should use icons according to the design. Please check if there is an icon in the `assets` folder. If not, `export` it from `figma`  https://www.figma.com/file/rRVxJpCu1G22D0jqyvK7I9/TeamMail?node-id=2%3A9

**Comment by dab246** [comment]
Next time you should write your PR according to the following standards:
#### Branch name
-  `feature/tf-###-short_description`: Add new features
-  `fix/tf-###-short_description`: Fix bug
-  `hot-fix/tf-###-short_description`: Urgent bug fix
-  `improve/tf-###-short_description`: Improved feature

#### PR title
`TF-###-short_description`
#### Commit name
`TF-###-description`

File: lib/features/composer/presentation/composer_controller.dart [hunk 4]
```diff
@@ -578,9 +580,15 @@ class ComposerController extends BaseController {
...
 
+    final emailHeader = <EmailHeader>{};
+    if(hasRequestReadReceipt.value) {
+      emailHeader.add(EmailHeader('Disposition-Notification-To', getEmailAddressSender()));
+    }
+
     return Email(
       generateEmailId,
       mailboxIds: mailboxIds.isNotEmpty ? mailboxIds : null,
+      headers: emailHeader,
```
Comment (dab246) [inline lib/features/composer/presentation/composer_controller.dart:591]: remove it if you used `headerMdn`

**Review by dab246 — COMMENTED** [review]
Please provide video demo test with `OpenPaas`

**Comment by sherlockvn** [comment]
> LGTM. How about the email in the receiver? Please make the demo with openpaas to make sure it works well. Thanks

i try. But i don't know why i have this error everytime i send a mail.

**Review by dab246 — APPROVED** [review]
/lgtm

**Review by hoangdat — APPROVED** [review]

**Pull request merged**